src/content/chainlink-automation/tutorials/dynamic-nft-qs.mdx

```diff
@@ -0,0 +1,71 @@
+
+1. Once the contract is deployed, you should be able to see more information under the **Deployed Contracts** section. You must mint an NFT. In the field next to **safeMint** paste your wallet address. Then, click the orange button and follow the prompts to comfirm the transaction.
+
+1. After minting the NFT, you can look at the stage of the NFT based on the tokenURIs given in the contract. Enter `0` in the **flowerStage** field and click the blue button. You should see the stage of the NFT. Enter `0` in the **tokenURI** field and click the blue button to see the corresponding URI for the NFT.
+
+1. Copy the contract address and open the [Chainlink Automation Interface](https://automation.chain.link). Connect your wallet to the network and click the blue button **Register New Upkeep**. For the Trigger mechanism, select Custom logic. You will prompted to enter your contract address. Paste your address and click **Next**.
+
+1. You must fill out the prompts to finish registering upkeep. For the **Starting balance (LINK)**, enter 0.5 for this demo. After entering information, click **Register Upkeep**. It may take a few moments for this to complete.
+
+1. Navigate back to Remix. Under the **Deployed Contracts** section, enter `[]` in the **checkUpkeep** field and click the blue button. You may see the following text: `upkeepNeeded true`. This means that an upkeep will be performed.
+
+1. While waiting for the upkeep to be performed, copy the contract address to your clipboard and open OpenSea(https://testnets.opensea.io/). Paste your address in search bar at the top of the page. You should be directed to a page with the image of your minted NFT. Click the seed thumbnail to learn more about the NFT.
+
+1. Navigate back to Remix. Enter `[]` in the **checkUpkeep** field and click the blue button. If this returns `upkeepNeeded false` your upkeep should have been performed. Navigate to the [Chainlink Automation Interface](automation.chain.link) and select your upkeep to view more details and to confirm that an upkeep was performed. If `upkeepNeeded false` is not returned in Remix, you may have to wait a few minutes and run the **checkUpkeep** function.
+
+1. Once the upkeep has been performed, navigate to Remix and click **flowerStage** and **tokenURI** to see the most recent flower stage and corresponding URI. Navigate back to the information of your NFT on OpenSea. Click the button on the top right to refresh the metadata. You should see the NFT become a bloom.
+
+1. To see the NFT become a fully blossomed flower, you will have to wait another 30 seconds (or the amount of time you denoted when deploying your contract). After this interval, refresh the NFT's metadata again. You should see a fully blossomed flower. You have now completed the tutorial.
```

Comment:
Add a summary of what this quickstart accomplished.

Reply:
Added summary; resolved